menubar.sh

```diff
@@ -1,15 +1,47 @@
 #!/bin/bash
 
+OS=$(uname -s)
 SERVICE=growler.py
-OUTFILE=~/Library/Logs/SecurityGrowler.log
+OUTFILE=/tmp/SecurityGrowler.log
```

Comment:
What about `.local/share/SecurityGrowler`? most distros with a DE have support for `.local`